rest/taskana-rest-spring/src/test/java/pro/taskana/rest/WorkbasketDefinitionControllerIntTest.java

```diff
@@ -65,12 +71,7 @@ void resetDb() {
 
   @Test
   void testExportWorkbasketFromDomain() {
-    ResponseEntity<List<WorkbasketDefinitionResource>> response =
```

Comment:
This piece of code was called that often so I refactored it to an simple method

Reply:
the imortant part is that a rest call is simulated here. "getXX" is a bit confusing. 
Please rename this to something more meaningful.